package.json

```diff
@@ -1,22 +1,22 @@
 {
-  "name": "pattern-replace-loader",
-  "version": "1.0.4",
-  "description": "pattern replace loader for webpack 2",
+  "name": "improved-pattern-replace-loader",
+  "version": "1.0.2",
```

Comment:
Next version should be `1.0.5`

Reply:
Yes, I have enabled that maintainers can edit the PR. The package.json is only different because I created a fork of it.